examples/coappearances/coappearances.flatdata

```diff
@@ -129,5 +129,26 @@ archive Graph {
 
     // All strings contained in the data separated by '\0'.
     strings: raw_data;
+
+    // Optional archive containing calculated statistics.
+    @optional
+    calculated_data : archive CalculatedData;
+}
+
+struct Invariants {
```

Comment:
Invariant is kind of a weird name

Reply:
Usually, this is a property invariant under a specific transformation; here, graph isomorphism: https://en.wikipedia.org/wiki/Category:Graph_invariants.